tests/test-json-posts-controller.php

```diff
@@ -148,6 +148,16 @@ public function test_get_post_context_without_permisson() {
 		$this->assertErrorResponse( 'json_post_cannot_edit', $response, 403 );
 	}
 
+	public function test_get_post_with_password_without_permisson() {
+		$post_id = $this->factory->post->create( array(
+			'post_password' => 'always$inthebananastand',
+		) );
+		$request = new WP_JSON_Request( 'GET', sprintf( '/wp/posts/%d', $post_id ) );
+		$response = $this->server->dispatch( $request );
+
+		$this->assertErrorResponse( 'json_user_cannot_read', $response, 401 );
+	}
+
```

Comment:
@danielbachhuber We actually already had this covered in the `test_create_post_with_password()` method.